src/wp-includes/rest-api/class-wp-rest-server.php

```diff
@@ -1507,4 +1509,24 @@ public function get_headers( $server ) {
 
 		return $headers;
 	}
+
+	/**
+	 * Add current theme link to API response for users who have proper permissions.
+	 *
+	 * @since 5.7.0
+	 *
+	 * @param WP_REST_Response $response REST API response.
+	 *
+	 * @return WP_REST_Response REST API response.
+	 */
+	protected function maybe_add_current_theme_link( \WP_REST_Response $response ) {
+		if ( ! current_user_can( 'switch_themes' ) && ! current_user_can( 'manage_network_themes' ) ) {
+			return $response;
+		}
+
+		$theme = wp_get_theme();
+		$response->add_link( 'active_theme', rest_url( 'wp/v2/themes/' . $theme->get_stylesheet() ) );
```

Comment:
`active_theme` isn't a [registered link relation](https://www.iana.org/assignments/link-relations/link-relations.xhtml), so we need to namespace it. We use the `api.w.org` namespace. For example, `https://api.w.org/term`.

Reply:
@TimothyBJacobs this has been addressed 👍